jakartaee11/JakartaEE11ReleasePlan.md

```diff
@@ -48,20 +49,27 @@ Continue to make CDI the single component model used across all of EE by removin
 
-### API Source and Target Level
-If a component Specification is planning a Major or Minor version update for Jakarta EE 11, then the recommendation would be to recompile and distribute the specification’s APIs at lowest required of Java SE 17 and Java SE 21.
+### API Source Code `--release` Level
 
-**Note:** A component specification may be required to recompile to a higher Java SE level than it actually use depending on the specifications it depends on.
+For the Jakarta EE Platform (Platform, Web and Core), the Java compiler `--release` option is 17. For the component specs, the Jakarta EE Platform requires the Java compiler `--release` option is **at most** 17, but component specifications can decide on a lower level.
 
 ### TCK Source Level
-The TCK for specifications updated in EE 11 will be compiled at the Java SE 21 level.
+
+- Component spec TCKs and platform TCK must compile at Java 17 or less.
+- A compatible component impl must pass their component TCK when run under Java 17 or 21.
+   - To ratify a component specification, there must exist an implementation that passes on Java 17. There must also exist an implementation that passes on 21.
+   - These need not be the same implementation. There can be one implementation that passes on 17 and a different one that passes on 21.
+- A compatible platform impl must pass the platform TCK when run under 17 or 21.
+   - To ratify a platform specification, there must exist an implementation that passes on 17. There must also exist an implementation that passes on 21.
+   - These need not be the same implementation. There can be one implementation that passes on 17 and a different one that passes on 21.
 
 #### Signature Tests
 In Jakarta EE 9.1, the framework for performing API Signature tests was updated to make it easier to test Signatures across multiple Java levels.
 The framework and/or API tests may need to be tweaked as we continue to learn and experiment with future levels of Java.
 
 ### Compatible Implementation (CI) Source Level
-How a Compatible Implementation supports the Java SE 21 runtime (or above) will be left as a vendor-defined solution.
+
+The long-standing policy of considering specification API binaries, in Maven Central or anywhere else, as a non-normative convenience remains unchanged. The platform project is silent on this matter. But because the platform project does mandate a specific JDK requirement for compatible implementations passing the component or platform TCK, specification binaries are practically constrained to follow that mandate. See [TCK Source Level](#tck-source-level)
```

Comment:
I'm not clear on what 'follow that mandate' means here.  Is the idea that the "API Source Code --release Level" section specifies source code requirements, so here we're saying that when the spec publishes a binary it shouldn't somehow work around those requirements and build a binary at a binary level above 17?

Reply:
I also have a mental hurdle moving from the first 2 sentences to the last too. I think the first too are meant as a kind of qualifier to say the last two don't are not to be read as creating a requirement to publish to Maven Central etc.

Perhaps the 3rd sentence can be:

"However, because the platform project does mandate a specific JDK requirement for compatible implementations passing the component or platform TCK, if a component specification does publish an API binary, that binary is practically constrained to follow that mandate."

Reply:
Fixed in 7f264e0 . Thanks @bstansberry .